layouts/shortcodes/specs.html

```diff
@@ -0,0 +1,27 @@
+{{ $specs := site.Data.specs }}
+<table>
+  <thead>
+    <tr>
+      <th>
+        Version
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    {{ range $specs }}
+    <tr>
+      <td>
+        <a href="{{ .url }}" target="_blank">
+          {{ .version }}
+        </a>
+      </td>
+      <td>
+        {{ .description | markdownify }}
```

Comment:
Once you rename this file to `spec-table.md`, you won't need to use `| markdownify`.